src/core/flexible_hvac.jl

```diff
@@ -40,6 +40,18 @@ end
 """
     FlexibleHVAC
 
+`FlexibleHVAC` is an option REopt input with the following keys: 
+```julia
+    system_matrix::AbstractMatrix{Float64}  # N x N, with N states (temperatures in RC network)
+    input_matrix::AbstractMatrix{Float64}  # N x M, with M inputs
+    exogenous_inputs::AbstractMatrix{Float64}  # M x T, with T time steps
+    control_node::Int64
+    initial_temperatures::AbstractVector{Float64}
+    temperature_upper_bound_degC::Union{Real, Nothing}
+    temperature_lower_bound_degC::Union{Real, Nothing}
+    installed_cost::Float64
```

Comment:
@NLaws  I removed `bau_hvac::BAU_HVAC` from this doc string, assuming it cannot be a user input. Is that correct?

Reply:
Yes